app/controllers/account/pages_controller.rb

```diff
@@ -26,4 +27,8 @@ def resource
   def collection
     current_user.companies.find(params[:company_id]).pages.by_rating
   end
+
+  def current_company
```

Comment:
update https://github.com/Ruby-IF081/heroes_team_app/pull/109/files#diff-5fbc6024966caaf0a54efe28813715c9R27 to use ```current_company```